### DISCUSSION TITLE
Remise aujourd’hui? 28/6

### DISCUSSION THREAD

**Discussion by claraquintela** [582-41E-MA/projet-web-2-CollRen#43] [Remises]
Bonjour la dream team

J'ai vu dans le calendrier que nous avons une remise à faire aujourd'hui de le sprint 1. 

@NabilDjerroud la maquette graphique fait aussi partie de cette remise. 

@CollRen pourrais tu fait encore cette remise? Je peux m'occuper de la prochaine. 

Merci

**Comment by CollRen** [comment]
@NabilDjerroud Est-ce que tu peux t'occuper de la remise de la maquette Figma, je m'occupe de remmtre le code et la documentation (Lien Github, diagrame etc.)

Ça vous va?

Voici le texte de l'énoncé:

Remise du document comprenant le diagramme de la base de données et les maquettes du site (pour les grands, moyens et petits écrans) - plusieurs pages, ainsi que le code développé jusqu'à la date conformément au plan du projet. Une réunion obligatoire avec le groupe est prévue le jour suivant la livraison (mardi le 2 juillet à 9h)

**Reply by NabilDjerroud** [reply]
Bonsoir René, désolé de ne pas avoir répondu sur le champs, je viens tout juste de remettre le travail, j'ai mis le pdf remiis à Marcos ici

**Comment by claraquintela** [comment]
Merci, René.
Je m’occcupe de la prochaine remise.

De ma part, l’unique chose qui manque c’est l’auth. J’espère la faire avant
la reunion de mardi matin 🤞🏼

**Reply by CollRen** [reply]
Ok, veux-tu que j'essaie de travailler là-dessus de mon côté aussi ?

**Comment by claraquintela** [comment]
Seulement si tu a VRAIMENT envie :)
Mais tout est sous contrôle. Pas besoin de t’inquiéter. Je vais suivre la
doc que Maxime Lacasse nous a donné.
Le sprint 2 sera intense, peut etre il sera mieux d’avancer les taches…

**Reply by CollRen** [reply]
Ok je vais me concentrer sur le sprint2. 

Clara, il faudra ajouter la colonne 'token' à la table utilisateurs pour pouvoir faire des login logout.

**Comment by claraquintela** [comment]
Noté. Merci :)
Je travaille lundi pour compenser aujourd’hui.

Bonne finde!

**Reply by CollRen** [reply]
Aussi, sache que cette colonne token a déjà été ajouté dans notre api: http://prodrdm.com/api/utilisateurs

**Comment by claraquintela** [comment]
Excellent. Merci encore.

**Comment by CollRen** [comment]
Ah, ce n'est pas un courriel pour se connecter, seulement un nom d'utilisateur...

**Reply by claraquintela** [reply]
Ah compris. 

Nous aurons:
Nom
Prenom
Courriel

Et aussi nom utilisateur. 

C'est bien ça?

**Reply by CollRen** [reply]
mot_de_passe et nom_utilisateur sont ce que l'on vérifie. Et nom_utilisateur est une string, pas un courriel

**Comment by claraquintela** [comment]
Tu préfères comme ça?
Ça empêche que un autre Rene puisse utiliser ce nom

**Reply by CollRen** [reply]
Je pense que c'est dans le devis de Marcos qu'il y a un nom_utilisateur...

**Reply by CollRen** [reply]
Je pars pour la fin de semaine... À plus !!

**Comment by NabilDjerroud** [comment]
Désolé les gars pour la réponse tardive, je travaillais tard je veins de finaliser toutes mes maquettes et à la dernieres minutes que j'ai du ajouter les maquettes pour mobiles et tablette, elles ne sont pas super bien travaillées mais présentables, je vous remet le lien du pdf que j'ai remis à Marcos
[Uploading Guide de conception - Maquettes- Nabil-Clara-René_.pdf…]()
